### ISSUE TITLE
Vote Tracker

### ISSUE THREAD

**Issue by mkanwal** [mitar/council-app#24]
Vote tracker for each user that tells them the % of motions they have voted on.  

We will eventually make a bylaw change to automatically issue uncooperative fines to people who do not vote in 25+% of all motions.

**Comment by mitar** [comment]
I do not agree with this, I think there are better approaches to solve this. For example, support delegation of votes.

So let's start implementing this once we observe other things to not work.

Also, we should at the moment first focus to get everyone to even try the site.

**Comment by mkanwal** [comment]
hi yes, 
im just putting down all potential ideas here, for future consideration.

**Comment by mitar** [comment]
OK.
